examples/tutorials/pruning_bert_glue.py

```diff
@@ -159,11 +159,21 @@ def preprocess_function(examples):
 from datasets import load_metric
 from transformers.modeling_outputs import SequenceClassifierOutput
 
+from nni.common.version import torch_version_is_2
+
+
+if torch_version_is_2():
+    from torch.optim.lr_scheduler import LRScheduler
+    SCHEDULER = LRScheduler
+else:
+    from torch.optim.lr_scheduler import _LRScheduler
+    SCHEDULER = _LRScheduler
+
 
 def training(model: torch.nn.Module,
              optimizer: torch.optim.Optimizer,
              criterion: Callable[[torch.Tensor, torch.Tensor], torch.Tensor],
-             lr_scheduler: torch.optim.lr_scheduler._LRScheduler = None,
+             lr_scheduler: SCHEDULER = None, # type: ignore
```

Comment:
after modifying the tutorial, re-build doc is needed (`make en`).
and before `# type: ignore` prefer double space.

Reply:
and why need type ignore here?

Reply:
good suggestion, I will modify it.